lib/components/molecules/StickyHeader/StickyHeader.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React, { useState, useRef, useEffect } from 'react';
+import styled from 'styled-components';
+import { Props } from './types/index';
+import useHandleScroll from '../../../../utils/util';
+
+import styles from './StickyHeader.style';
+
+const StickyHeader = ({ children, className }: Props): Node => {
+  const [isSticky, setSticky] = useState(false);
+  const wrapperRef = useRef(null);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => useHandleScroll(wrapperRef, setSticky));
+    return () => {
+      window.removeEventListener('scroll', useHandleScroll);
+    };
+  });
+
+  return (
+    <div aria-label="Sticky Header" className={`${isSticky ? className : ''}`} ref={wrapperRef}>
+      <header>{children}</header>
```

Comment:
@Vipul6 I don't think you have fixed the original problem i had highlighted. This component is causing a flicker when it gets sticky and also hiding the page content below itself.

Reply:
Fixed.